azurelinuxagent/ga/exthandlers.py

```diff
@@ -276,7 +273,7 @@ def run(self):
                       message=detailed_msg)
             return
 
-    def cleanup_outdated_handlers(self):
+    def _cleanup_outdated_handlers(self):
```

Comment:
Made those private because i was refactoring these methods; I abandoned the refactoring, but these should stay private.